net/udprelay/server.go

```diff
@@ -880,3 +877,12 @@ func (s *Server) getDERPMap() *tailcfg.DERPMap {
 	defer s.mu.Unlock()
 	return s.derpMap
 }
+
+// SetStaticAddrPorts sets addr:port pairs the [Server] will advertise
+// as candidates it is potentially reachable over, in combination with
+// dynamically discovered pairs.
```

Comment:
Perhaps note also that calling this replaces any previously-provided static values, rather than adding to them.